ironfish/src/network/utils/parseUrl.ts

```diff
@@ -2,6 +2,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
+export type WebSocketAddress = {
```

Comment:
We should consider changing the filename since it is no longer specific to the parseUrl function

Reply:
agree, I'll make that change in this PR